components/brave_rewards/browser/publisher_info_database.cc

```diff
@@ -7,6 +7,7 @@
 #include <stdint.h>
 
 #include <string>
+#include <bat/ledger/pending_contribution.h>
```

Comment:
`#include "bat/ledger/pending_contribution.h"` ?
and this line seems unnecessary since it is included in publisher_info_database.h already.
Or maybe do forward declaration in publisher_info_database.h directly then include the header here.

Reply:
this was automatically added by my editor, removed